go/vt/vtgate/evalengine/integration/string_fun_test.go

```diff
@@ -204,3 +204,51 @@ func TestBuiltinRepeat(t *testing.T) {
 
 	}
 }
+
+func TestBuiltinConv(t *testing.T) {
+	var conn = mysqlconn(t)
+	defer conn.Close()
+	cases := []string{
+		"-5.1",
+		"-5.9",
+		"0xa21 + '1'",
+		"-0xa21 + '1'",
+		"10",
+		"10 + '10' + 10",
+		"10 + '10' - 10",
+		"-10",
+		"'10'",
+		"10+'10'+'10a'+X'0a'",
+		"10 / 10",
+		"X'0FFFFFFFFFFFFFF'",
+		"99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999",
+		"-99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999",
+	}
```

Comment:
These still aren't strong enough cases. What happens if we have a `+` in the beginning? What if we have 2 of those in the beginning? What if it is a string with a + and not a number?